docs/source/tutorials/Index.md

```diff
@@ -0,0 +1,14 @@
+Index.md
+
+All MAAP data tutorials can be found in this folder. 
+
+Contents:
+1. GEDI
+	a. L1A
+	b. L1B
+	c. L2A 
+	d. L2B
+		In this tutorial we use the integrated Earthdata serch function in MAAP to search for some GEDI L2B data for an area of interest. The data is then downloaded and prepared for analysis. We perform a spatial subset on the downloaded data to reduce data volumes and create some basic plots.
```

Comment:
Want to format this so it's not all on one line?

Reply:
Done. Can you check?

Reply:
Just from eyeing it, the line format looks good. Is the text for one of the examples? Should it be in a nested list or is it describing all L2B examples?